app/controllers/api/internal/v1/laa_references_controller.rb

```diff
@@ -8,9 +8,12 @@ def create
           contract = NewLaaReferenceContract.new.call(**transformed_params)
           enforce_contract!(contract)
 
-          LaaReferenceCreator.call(defendant_id: transformed_params[:defendant_id],
-                                   user_name: transformed_params[:user_name],
-                                   maat_reference: transformed_params[:maat_reference])
+          MaatLinkCreatorWorker.perform_async(
```

Comment:
Is there a reason these aren't named params?

Reply:
Tried with named params, but doesn't work.

It's the way Sidekiq's `perform_async` method is implemented: https://github.com/mperham/sidekiq/blob/3330df0ee37cfd3e0cd3ef01e3e66b584b99d488/lib/sidekiq/worker.rb#L163